src/modules/ToDoList.js

```diff
@@ -0,0 +1,68 @@
+class ToDoList {
+  #toDoList;
+
+  constructor() {
+    const storedToDos = localStorage.getItem('to-do-list');
+    this.#toDoList = storedToDos ? JSON.parse(storedToDos) : [];
+  }
+
+  get toDoList() {
+    return this.#toDoList;
+  }
+
+  createToDoItem(description) {
+    const newToDo = {
+      index: this.#toDoList.length,
+      description,
+      completed: false,
+    };
```

Comment:
- [ ] It is expected that the `index` property of the to-do object is set to the length of the value of the new array length. Therefore, if you're adding a 5th task to the list, the index of that task should equal to 5. Attached below is a `localStorage` screenshot of your application showing that the indexes don't satisfy this requirement. Please implement this project requirement 👍 
![image](https://github.com/henokkhm/microverse-webpack-to-do-list/assets/46839548/5658ac4a-8603-4f1e-8d8c-35123760c0de)